internal/business/persisted_operations/persisted_operations.go

```diff
@@ -274,7 +279,16 @@ func (p *PersistedOperationsHandler) reloadFromRemote() {
 	ctx, cancel := context.WithTimeout(ctx, p.cfg.Reload.Timeout)
 	defer cancel()
 
-	err := p.remoteLoader.Load(ctx)
+	startTime := time.Now()
+
+	err := p.remoteLoader.Load(ctx, p.log)
+
+	endTime := time.Since(startTime).Seconds()
+
+	p.log.Info(fmt.Sprintf("Loading files from bucket took: %f seconds", endTime))
+
+	reloadGauge.WithLabelValues("remote", "downloadDuration").Set(endTime)
```

Comment:
isnt a measure on time mostly relevant if you can relate it to the count of objects downloaded?

Reply:
yes and no, the time it takes to load the persisted operations files is the time we're not ready yet to serve traffic on startup which can be an interesting thing on its own to monitor. Since downloading seems to be the most time consuming part i figured i'd put a metric there

Reply:
i've also exposed number of files read as a metric so we can combine the 2 into a nice dashboard

Reply:
maybe the both can be combined into a single metric?